docs/proposals/Network-Isolation.md

```diff
@@ -0,0 +1,72 @@
+# Network Isolation
+
+## Introduction
+This [redhat/ovs-multitenant network plugin](https://docs.openshift.com/container-platform/3.6/architecture/additional_concepts/sdn.html#architecture-additional-concepts-sdn) will restrict namespaces from having network traffic flow between them. 
+
+> The ovs-multitenant plug-in provides OpenShift Container Platform project level isolation for pods and services. Each project receives a unique Virtual Network ID (VNID) that identifies traffic from pods assigned to the project. Pods from different projects cannot send packets to or receive packets from pods and services of a different project.
+
+The ASB creates a transient namespace while running an APB and grants the correct access to the target namespace. This network plugin will cause APBs that assume they can reach, over the network the pod in the target namespace, to fail. 
```

Comment:
Let's try to reword that last sentence.

> This network plugin will cause some APBs to fail, if the APB assumes it can access pods in the target namespace over the network.